modules/nf-core/thermorawfileparser/tests/main.nf.test

```diff
@@ -0,0 +1,59 @@
+nextflow_process {
+
+    name "Test Process THERMORAWFILEPARSER"
+    script "../main.nf"
+    process "THERMORAWFILEPARSER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "thermorawfileparser"
+
+    test("proteomics - parse - raw") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test'], // meta map
+                    file(params.test_data['proteomics']['msspectra']['hla2_file'], checkIfExists: true)
```

Comment:
Think some were restructuring the input data, will have to double check if it's become standard practice. 
```suggestion
                    file(params.modules_testdata_base_path +'/proteomics/msspectra/PXD012083_e005640_II.raw', checkIfExists: true)
```

Reply:
Oh, that is new to me.. Can you provide me the slack link to the conversation? :)

If it is not standard yet, I would keep it for now as is if you don't mind

Reply:
It's been scattered across a couple of threads. It comes down to the being able to test pipelines in offline settings. 
https://nfcore.slack.com/archives/CJRH30T6V/p1704822277079199?thread_ts=1704755682.903089&cid=CJRH30T6V
https://nfcore.slack.com/archives/CJRH30T6V/p1704822537591969
It's your module I'll leave you up to decide if you want to change it or not.

Reply:
Thanks for the pointers! I would keep it as it is for now and update all modules I created as soon as a final decision has been done.